Automattic/woocommerce-connect-client
# Upgrade to node 5.11.1

This PR seeks to bump the recommended/tested version of Node to 5.11.1 to match up with Calypso.

* change the travis version
* update the readme
* add an [nvmrc](https://github.com/creationix/nvm#nvmrc)

To test:

* Upgrade your local install of node to 5.11.1 (e.g. via nvm)
* Run `npm test` make sure all tests pass
* Run `npm start` make sure the app runs (e.g. settings screens load)
* Run `npm run dist` make sure the app compiles

cc @allendav @jeffstieler @nabsul
